tools/anthosbm-ansible-module/roles/shared-files/cluster_config.yaml.j2

```diff
@@ -67,6 +67,7 @@ spec:
 {% if (cluster_type != 'admin') %}
       ingressVIP: {{ ingress_vip }}
 {% if lb_mode is defined and lb_mode == 'bundled' %}
+    addressPools:
 {% for lb_address in lb_address_pool %}
     addressPools:
```

Comment:
this line is not needed. 

Please also at least test this out by generating a correct cluster config file

Reply:
Done - tested internally that

 * multiple `address_pool` do generate multiple address pools for LB
 * single `address_pool` do generate only one address pool for LB
 * no `address_pool` do return error
 * manual lb type does not generate address pool